### PR TITLE
Fix bc data

### PR DESCRIPTION
**Pull request by corpaul**
actually record the right bartercast stuff :)
+ advance dispersy pointer for the isolated network gumby stuff

### CODE REVIEW

**Comment by corpaul** [comment]
retest this please

**Comment by rjruigrok** [comment]
@corpaul There was a bug in the TunnelCommunity which is fixed in the PR that was merged 20 minutes ago to devel. :)

**Comment by corpaul** [comment]
retest this please

**Comment by corpaul** [comment]
retest this please

**Comment by whirm** [comment]
@corpaul rebase :)

**Comment by corpaul** [comment]
retest this please

**Comment by corpaul** [comment]
retest this please

**Comment by corpaul** [comment]
yey, ready to merge

File: Tribler/community/tunnel/tunnel_community.py [hunk 1]
```diff
@@ -1225,32 +1226,27 @@ def increase_bytes_sent(self, obj, num_bytes):
...
             self.stats['bytes_relay_up'] += num_bytes
-            _barter_statistics.dict_inc_bartercast(BartercastStatisticTypes.TUNNELS_RELAY_BYTES_SENT,
-                                                   obj.mid, num_bytes)
+            _barter_statistics.dict_inc_bartercast(BartercastStatisticTypes.TUNNELS_RELAY_BYTES_SENT, "%s:%s" % (obj.sock_addr[0], obj.sock_addr[1]), num_bytes)
         elif isinstance(obj, TunnelExitSocket):
             obj.bytes_up += num_bytes
             self.stats['bytes_exit'] += num_bytes
-            _barter_statistics.dict_inc_bartercast(BartercastStatisticTypes.TUNNELS_EXIT_BYTES_SENT,
-                                                   obj.mid, num_bytes)
+            # todo check if obj.mid is the right mid!
```
Comment (whirm) [inline Tribler/community/tunnel/tunnel_community.py]: Are you planning on fixing this one on another PR or you overlooked it?

**Comment by corpaul** [comment]
oops forgot to remove that comment.. i don't use obj.mid for the statistics anymore

**Pull request merged**